pinot-core/src/main/java/org/apache/pinot/core/operator/blocks/TransformBlock.java

```diff
@@ -78,4 +79,54 @@ public BlockDocIdValueSet getBlockDocIdValueSet() {
   public BlockMetadata getMetadata() {
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  public void pushDown(String column, TransformEvaluator evaluator, int[] buffer) {
+    _projectionBlock.pushDown(column, evaluator, buffer);
```

Comment:
any need to check `if (expression.getType() == ExpressionContext.Type.IDENTIFIER)` like L51 above? iiuc, we pushDown only when there is no transforms, i.e. a pass-through to projection block.

Reply:
If there is a column name it must be an identifier. I had previously only implemented this for `ProjectionBlock` for this reason though.